README.md

```diff
@@ -290,7 +291,14 @@ To run the development server:
 bin/server
 ```
 
+# Other projects
 
+Not your cup of tea? 🍵 Here are some other Elixir alternatives we like:
```

Comment:
This teacup has no handle. The tea is hot. I'm against this emoji

won't block though

Reply:
☕ is the one with the handle 😞